src/snowflake/snowpark/modin/plugin/extensions/datetime_index.py

```diff
@@ -1025,8 +1024,13 @@ def tz_convert(self, tz) -> DatetimeIndex:
                        '2014-08-01 09:00:00'],
                         dtype='datetime64[ns]', freq='h')
         """
+        return DatetimeIndex(
```

Comment:
Should we unskip all related doctests?

Reply:
Unfortunately, most of them cannot be unskipped yet because `date_range` does not support `tz` yet. I added a TODO.

Reply:
I also unskipped the ones where this was possible for `tz_localize`.